test/test_samples_osw/base_results/results.csv

```diff
@@ -1,9 +1,19 @@
icity_pumps_cooling_kwh,electricity_pumps_heating_kwh,electricity_pv_kwh,electricity_range_fan_kwh,electricity_recirc_pump_kwh,electricity_refrigerator_kwh,electricity_vehicle_kwh,electricity_water_systems_kwh,electricity_well_pump_kwh,floor_area_attic_ft_2,floor_area_conditioned_ft_2,floor_area_lighting_ft_2,fuel_oil_heating_mbtu,fuel_oil_water_systems_mbtu,geometry_attic_type,geometry_building_horizontal_location_mf,geometry_building_horizontal_location_sfa,geometry_building_level_mf,geometry_building_number_units_mf,geometry_building_number_units_sfa,geometry_building_type_acs,geometry_building_type_height,geometry_building_type_recs,geometry_floor_area,geometry_floor_area_bin,geometry_foundation_type,geometry_garage,geometry_stories,geometry_stories_low_rise,geometry_wall_type,heating_fuel,heating_setpoint,heating_setpoint_has_offset,heating_setpoint_offset_magnitude,heating_setpoint_offset_period,holiday_lighting,hot_water_distribution,hot_water_fixtures,hours_cooling_setpoint_not_met,hours_heating_setpoint_not_met,hvac_cooling_capacity_w,hvac_cooling_efficiency,hvac_cooling_type,hvac_has_ducts,hvac_has_shared_system,hvac_heating_capacity_w,hvac_heating_efficiency,hvac_heating_supp_capacity_w,hvac_heating_type,hvac_heating_type_and_fuel,hvac_shared_efficiencies,include_enduse_subcategories,infiltration,insulation_crawlspace,insulation_finished_basement,insulation_finished_roof,insulation_interzonal_floor,insulation_pier_beam,insulation_slab,insulation_unfinished_attic,insulation_unfinished_basement,insulation_wall,interior_shading,lighting,lighting_interior_use,lighting_other_use,location_city,location_latitude,location_longitude,mechanical_ventilation,misc_extra_refrigerator,misc_freezer,misc_gas_fireplace,misc_gas_grill,misc_gas_lighting,misc_hot_tub_spa,misc_pool,misc_pool_heater,misc_pool_pump,misc_pool_schedule,misc_well_pump,natural_gas_clothes_dryer_therm,natural_gas_cooking_range_therm,natural_gas_fireplace_therm,natural_gas_grill_therm,natural_gas_heating_therm,natural_gas_hot_tub_heater_therm,natural_gas_lighting_therm,natural_gas_pool_heater_therm,natural_gas_water_systems_therm,natural_ventilation,neighbors,occupants,option_01_cost_usd,option_01_lifetime_yrs,orientation,overhangs,plug_load_diversity,plug_loads,propane_clothes_dryer_mbtu,propane_cooking_range_mbtu,propane_heating_mbtu,propane_water_systems_mbtu,pv,radiant_barrier,range_spot_vent_hour,refrigerator,refrigerator_schedule,roof_area_ft_2,roof_material_finished_roof,roof_material_unfinished_attic,setpoint_demand_response,size_cooling_system_kbtu_h,size_heating_system_kbtu_h,size_water_heater_gal,solar_hot_water,state,total_site_electricity_kwh,total_site_energy_mbtu,total_site_fuel_oil_mbtu,total_site_natural_gas_therm,total_site_propane_mbtu,total_site_wood_mbtu,vacancy_status,wall_area_above_grade_conditioned_ft_2,wall_area_above_grade_exterior_ft_2,wall_area_below_grade_ft_2,water_heater_efficiency,window_area_ft_2,window_areas,windows,wood_heating_mbtu,workflow_json
```

Comment:
@joseph-robertson I think having two columns that define the run will break your compare script (which assumes there is only one header column)?

Reply:
Especially since the first column has duplicates (e.g., 7.osw below).

Reply:
Good point. Instead of a second column with project name, I'll instead prefix the osw string with the project name.